client/components/buttons/iconButtons/showMoreIconButton.tsx

```diff
@@ -0,0 +1,85 @@
+import { IconButton, Menu, MenuItem, Tooltip } from "@mui/material";
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
+import React, { useState } from 'react';
+import axios from "axios";
+import Album from "@/types/albumTypes";
+import Artist from "@/types/artistTypes";
+import Playlist from "@/types/playlistTypes";
+import Track from "@/types/trackTypes";
+
+interface Props {
+    token: string
+    item: Track | Album | Playlist | Artist
+    handleAdding: (newAdd: Track | Album | Playlist | Artist) => void
+    enableAddButton: () => void
+}
+
+export default function ShowMoreIconButton({
+    token,
+    item,
+    handleAdding,
+    enableAddButton
+}: Props) {
+    const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
```

Comment:
I would avoid using using abbreviations in variables

Reply:
Not this time